company_website/static/main_page/technologies.sass

```diff
@@ -0,0 +1,28 @@
+@import "partials/constants"
+
+.container
+	max-width: unset
+	padding: 0 $container-padding
+	@media only screen and (max-width: $md-screen-size)
+		padding: 0 10%
+
+	.row.tech-img
+		margin: 0 auto
+
+		.technology
+			width: 4.7vw
+			min-width: 70px
+			margin: 20px auto
+
+			@media only screen and (max-width: $md-screen-size)
+				height: 120px
+				width: auto
+
+.nopadding
+	padding: 0
+	margin: 0
+
+.hr-line
```

Comment:
Please move `nopadding and `hr-line` classes to main class -> `container`. You are using this class inside container class anyway